blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -514,44 +489,21 @@ private static void createAndInsertTestEvents(final String tenant, int eventCoun
         eventsSearchIO.insert(tenant, eventList);
     }
 
-    private HttpResponse postEvent(String requestBody, String tenantId) throws Exception {
-        URIBuilder builder = new URIBuilder().setScheme("http").setHost("127.0.0.1")
-                .setPort(httpPort).setPath("/v2.0/" + tenantId + "/events");
-        HttpPost post = new HttpPost(builder.build());
-        HttpEntity entity = new StringEntity(requestBody,
-                ContentType.APPLICATION_JSON);
-        post.setEntity(entity);
-        post.setHeader(Event.FieldLabels.tenantId.name(), tenantId);
-        HttpResponse response = client.execute(post);
-        return response;
+    private String createTestEvent( int batchSize ) throws Exception {
+
+        return createTestEvent( batchSize, System.currentTimeMillis() );
     }
 
-    private static String createTestEvent(int batchSize) throws Exception {
+    private String createTestEvent(int batchSize, long timestamp) throws Exception {
         StringBuilder events = new StringBuilder();
         for (int i=0; i<batchSize; i++) {
             Event event = new Event();
             event.setWhat("deployment "+i);
-            event.setWhen(Calendar.getInstance().getTimeInMillis());
+            event.setWhen( timestamp );
             event.setData("deploying prod "+i);
             event.setTags("deployment "+i);
             events.append(new ObjectMapper().writeValueAsString(event));
         }
         return events.toString();
     }
-    
-    @AfterClass
-    public static void shutdown() {
```

Comment:
Previously this class didn't subclass ```HttpIntegrationTestBase```. The base class does a lot of the init & shut down that this class was doing.  I also wanted to leverage some other commonalities, so I extended ```HttpIntegrationTestBase``` and removed the duplicate code.